Dockerfile

```diff
@@ -1,13 +1,16 @@
-FROM nvidia/cuda:8.0-devel-ubuntu16.04
+#
+#Builds and installs RAJA using the gcc8 compiler
+#
+
+FROM rajaorg/compiler:gcc8
 MAINTAINER RAJA Development Team <raja-dev@llnl.gov>
 
-RUN apt-get update -y
-RUN apt-get install -y git cmake gdb
+COPY --chown=raja:raja . /home/raja/src
 
-RUN cd /opt/ && git clone https://github.com/LLNL/RAJA.git
+WORKDIR /home/raja/src
 
-WORKDIR /opt/RAJA
+RUN  mkdir build && cd build && cmake -DENABLE_CUDA=OFF ..
 
-RUN mkdir build && cd build && cmake -DENABLE_CUDA=ON ..
+RUN cd build && sudo make -j 3 && sudo make install
 
-RUN cd build && make -j && make install
+CMD ["bash"]
```

Comment:
@artv3 - can you verify which directory this drops into? I'm assuming `~/`, but if not we might need to update the tutorial accordingly.

Reply:
It drops you at /home/raja/src

Reply:
Okay, I think that works best for the tutorial right?

Reply:
It might be good to change the src folder name to work-space or something like that.  
As is /home/raja/src contains the raja files and folders, one of them is being src.

Reply:
Sure, that would make sense.